index.yaml

```diff
@@ -33,52 +33,20 @@ entries:
     urls:
     - assets/ambassador/ambassador-6.7.1100+up6.7.11.tgz
     version: 6.7.1100+up6.7.11
-  - annotations:
```

Comment:
Charts like this that were added but never released are dropped from the annotations since they do not need to exist